src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -1517,6 +1528,10 @@ static Library()
                 BuiltinFunctionsCore.BooleanN_T,
                 StandardErrorHandlingTabularOverload<NumberValue>(BuiltinFunctionsCore.BooleanN_T.Name, SimpleFunctionImplementations[BuiltinFunctionsCore.BooleanN])
             },
+            {
+                BuiltinFunctionsCore.BooleanB_T,
+                StandardErrorHandlingTabularOverload<NumberValue>(BuiltinFunctionsCore.BooleanB_T.Name, SimpleFunctionImplementations[BuiltinFunctionsCore.BooleanB])
```

Comment:
>NumberValue [](http://example.com/codeflow?start=53&length=11)

BooleanValue

Reply:
Since we don't have Implementation of TextToBoolean with BooleanValue, for consistency updated to StringValue as this is actually not used but rather a placeholder.